src/elements/content-uploader/ItemRemove.tsx

```diff
@@ -29,20 +30,13 @@ const ItemRemove = ({ onClick, status }: ItemRemoveProps) => {
         resin['data-resin-target'] = target;
     }
 
-    const intl = useIntl();
     const isDisabled = status === STATUS_STAGED;
-    const tooltipText = intl.formatMessage(messages.remove);
+    const tooltipText = formatMessage(messages.remove);
 
     return (
         <div className="bcu-item-action">
-            <Tooltip content={tooltipText} variant="standard">
-                <IconButton
-                    aria-label={tooltipText}
-                    disabled={isDisabled}
-                    onClick={onClick}
-                    icon={() => <XMark color={GrayBlack} height={Size5} width={Size5} />}
-                    {...resin}
-                />
+            <Tooltip content={tooltipText}>
+                <IconButton aria-label={tooltipText} disabled={isDisabled} onClick={onClick} icon={XMark} {...resin} />
```

Comment:
Just want to make sure that it is intended to use the default icon styles here while overriding the prop like `color` for XMark?

Reply:
good question. yes this is intended since `IconButton` has styles that will handle different states like hovering. if we override `color` than the styles wouldn't match the rest of the design

in the case of `IconInProgress`, this is a custom component that handles the loading state together with the icon. but it should probably be refactored because i don't think it's good for accessibility